PED/SetPedShouldPlayDirectedScenarioExit.md

```diff
@@ -6,13 +6,19 @@ aliases: ["0xEC6935EBE0847B90"]
 
 ```c
 // 0xEC6935EBE0847B90 0x761F8F48
-Any _SET_PED_SHOULD_PLAY_DIRECTED_SCENARIO_EXIT(Any p0, Any p1, Any p2, Any p3);
+BOOL _SET_PED_SHOULD_PLAY_DIRECTED_SCENARIO_EXIT(PED_INDEX pedIndex, float x, float y, float z);
```

Comment:
Hey this will error, `PED_INDEX` isn't defined. Same with all the other `Any` to `float`. `cs_type` is needed.

i.e.
```c
cs_type(Any) BOOL _SET_PED_SHOULD_PLAY_DIRECTED_SCENARIO_EXIT(Ped ped, cs_type(Any) float x, cs_type(Any) float y, cs_type(Any) float z);
```

I will fix this for you anyway.